CMakeLists.txt

```diff
@@ -146,7 +122,7 @@ if(CAMODOCAL_PLATFORM_UNIX)
     #set_linker_flags("-Wl,--no-undefined -ldl -lnsl -lm -pthread -lrt" CACHE)
     find_package(LibDL)
     find_package(Libnsl)
-    find_package(LibM)
+    # find_package(LibM)
```

Comment:
Is libM no longer required? Also we have to be careful about capitalization for case sensitive/unsensitive filesystems.